1.0/18-alpine3.17/docker-entrypoint.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -eo pipefail
+
+# if command does not start with mongo-express, run the command instead of the entrypoint
+if [ "${1}" != "mongo-express" ]; then
+    exec "$@"
+fi
+
+function wait_tcp_port {
+    local host="$1" port="$2"
+    local max_tries=5 tries=1
+
+    # see http://tldp.org/LDP/abs/html/devref1.html for description of this syntax.
```

Comment:
Neat, I haven't seen something like this before. I had chatgpt help me out a bit to understand this too: 

https://chat.openai.com/share/eca50d54-a1f4-4771-94fe-a8c3063f7676